test/core/transport/chttp2/http2_client_transport_test.cc

```diff
@@ -0,0 +1,21 @@
+//
+//
+// Copyright 2024 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+// TODO(tjagtap) : [PH2][P1] : Write these tests once the fist draft of
+// http2_client_transport and http2_server_transport is submitted.
+// And the build changes will be added after the basic NOOP test is running.
```

Comment:
Please add a test that Http2ClientTransport can be instantiated as part of this PR.

Reply:
(and a BUILD target so it's running in CI)

Reply:
Done